src/Extensions/Subsites/IndexConfigurationExtension.php

```diff
@@ -15,14 +15,42 @@ class IndexConfigurationExtension extends Extension
     public function updateIndexesForDocument(DocumentInterface $doc, array &$indexes): void
     {
         $docSubsiteId = null;
+        /** @var DataObject $doc  */
```

Comment:
the $doc->**getDataObject**() part is sending an error if it is the `$dataObj`

Reply:
Interesting - because `$doc` isn't a `DataObject`, it's a `DataObjectDocument` I believe? To be honest you can probably remove that line entirely, it's only a typehint